File: product_code_4.js

function calcularEnvio(pais, totalCompra) {
    if (pais.toLowerCase() === "méxico" && totalCompra > 100) {
        return "¡Envío gratis a todo México!";
    } else {
        return "Costo de envío: 8 USDT";
    }
}

console.log(calcularEnvio("México", 120));

File: product_code_4.py

def calcular_envio(pais, total_compra):
    if pais.lower() == "méxico" and total_compra > 100:
        return "¡Envío gratis a todo México!"
    else:
        return "Costo de envío: 8 USDT"

print(calcular_envio("México", 120))
